### DISCUSSION TITLE
rabbitmq3.8.x how to change DirtyIOSchedulers

### DISCUSSION THREAD

**Discussion by jiangxinlingdu** [rabbitmq/rabbitmq-server#3070] [Other] [answered]
i see :https://erlang.org/doc/man/erl.html#+SDio

+SDio DirtyIOSchedulers
Sets the number of dirty I/O scheduler threads to create. Valid range is 1-1024. By default, the number of dirty I/O scheduler threads created is 10.

The amount of dirty IO schedulers is not limited by the amount of normal schedulers like the amount of dirty CPU schedulers. This since only I/O bound work is expected to execute on dirty I/O schedulers. If the user should schedule CPU bound jobs on dirty I/O schedulers, these jobs might starve ordinary jobs executing on ordinary schedulers.

Typical users of the dirty IO schedulers are reading and writing to files.

You can use msacc(3) in order to see the current load of the dirty IO schedulers threads and adjust the number used accordingly.

i set :
![image](https://user-images.githubusercontent.com/6373972/119972535-e9269280-bfe4-11eb-89ea-d4145a23396e.png)
![image](https://user-images.githubusercontent.com/6373972/119972571-f3489100-bfe4-11eb-8d9c-cdadb6035ac8.png)
![image](https://user-images.githubusercontent.com/6373972/119972617-00658000-bfe5-11eb-852c-80e214be6ba2.png)

can you tell me how to change DirtyIOSchedulers

**Comment by michaelklishin** [comment]
I will convert this issue to a GitHub discussion. Currently GitHub will [automatically close and lock the issue](https://github.com/github/feedback/discussions/3197) even though your question will be transferred and responded to elsewhere. This is to let you know that we do not intend to ignore this but this is how the current GitHub conversion mechanism makes it seem for the users :(

**Comment by michaelklishin** [comment, marked as answer]
The same way you'd [configure any other Erlang VM flags for RabbitMQ](https://www.rabbitmq.com/runtime.html#vm-settings).
